### DISCUSSION TITLE
Issue with "filter posts by tags" trick

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1602] [Help] [answered]
Hi, I noticed, that on pages which uses the filter trick #1465 a kind of a loading circle appears. I am not sure whether was from the beginning, but maybe somebody has anode, how to remove this thing. Thanks!

![circle](https://github.com/user-attachments/assets/5740338f-e740-45fc-a7c9-9ad93b4cad76)
can be seen on https://fluegge.net/unterwegs/mallorca/

**Comment by michaelfluegge** [comment, marked as answer]
I've found it :-) 
It is a CSS issue with .l-col:before . I set it to none 
 .l-col:before {
    display: none; }
